multilabel_confusion_matrix_test.py is not run by bazel

I am Tensorflow Addons' worst nightmare! Bringer of bad news! 

**Describe the bug**

multilabel_confusion_matrix_test.py is missing:

```
if __name__ == "__main__":
    tf.test.main()
```

A test is failing in this file, but since bazel didn't run it, we never knew.

**Other info / logs**

```
(proof_of_concept_pytest) /mnt/c/Users/gdemarmi/Desktop/projects/addons $ pytest -v tensorflow_addons/metrics/multilabel_confusion_matrix_test.py
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
======================================================================== test session starts ========================================================================
platform linux -- Python 3.7.5, pytest-5.0.1, py-1.8.0, pluggy-0.13.0 -- /home/gdemarmi/softwares/python/anaconda/bin/python
cachedir: .pytest_cache
rootdir: /mnt/c/Users/gdemarmi/Desktop/projects/addons
plugins: arraydiff-0.3, cov-2.7.1, doctestplus-0.2.0, flake8-1.0.4, forked-1.0.2, openfiles-0.4.0, pep8-1.0.6, remotedata-0.3.2, xdist-1.27.0, typeguard-2.7.1
collected 5 items

tensorflow_addons/metrics/multilabel_confusion_matrix_test.py::MultiLabelConfusionMatrixTest::test_config FAILED                                              [ 20%]
tensorflow_addons/metrics/multilabel_confusion_matrix_test.py::MultiLabelConfusionMatrixTest::test_mcm_3_classes PASSED                                       [ 40%]
tensorflow_addons/metrics/multilabel_confusion_matrix_test.py::MultiLabelConfusionMatrixTest::test_mcm_4_classes PASSED                                       [ 60%]
tensorflow_addons/metrics/multilabel_confusion_matrix_test.py::MultiLabelConfusionMatrixTest::test_multiclass PASSED                                          [ 80%]
tensorflow_addons/metrics/multilabel_confusion_matrix_test.py::MultiLabelConfusionMatrixTest::test_session SKIPPED                                            [100%]

============================================================================= FAILURES ==============================================================================
_____________________________________________________________ MultiLabelConfusionMatrixTest.test_config _____________________________________________________________

self = <tensorflow_addons.metrics.multilabel_confusion_matrix_test.MultiLabelConfusionMatrixTest testMethod=test_config>, args = (), kwargs = {}

    def decorated(self, *args, **kwargs):
      try:
        with context.graph_mode():
          with self.test_session(use_gpu=use_gpu, config=config):
>           f(self, *args, **kwargs)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1111:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tensorflow_addons/metrics/multilabel_confusion_matrix_test.py:27: in test_config
    self.assertEqual(mcm_obj.dtype, tf.int32)
E   AssertionError: 'float32' != tf.int32
----------------------------------------------------------------------- Captured stderr call ------------------------------------------------------------------------
WARNING: Logging before flag parsing goes to stderr.
W0302 15:52:27.043615 140538783270720 deprecation.py:323] From /home/gdemarmi/softwares/python/anaconda/lib/python3.7/contextlib.py:82: TensorFlowTestCase.test_session (from tensorflow.python.framework.test_util) is deprecated and will be removed in a future version.
Instructions for updating:
Use `self.session()` or `self.cached_session()` instead.
2020-03-02 15:52:27.048341: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2020-03-02 15:52:27.048525: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2020-03-02 15:52:27.048619: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (CRE1-L11685): /proc/driver/nvidia/version does not exist
2020-03-02 15:52:27.050524: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-03-02 15:52:27.067775: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2712000000 Hz
2020-03-02 15:52:27.069353: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7fffed609bc0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-03-02 15:52:27.069507: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
W0302 15:52:27.084591 140538783270720 deprecation.py:506] From /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
------------------------------------------------------------------------- Captured log call -------------------------------------------------------------------------
WARNING  tensorflow:deprecation.py:323 From /home/gdemarmi/softwares/python/anaconda/lib/python3.7/contextlib.py:82: TensorFlowTestCase.test_session (from tensorflow.python.framework.test_util) is deprecated and will be removed in a future version.
                                       Instructions for updating:
                                       Use `self.session()` or `self.cached_session()` instead.
WARNING  tensorflow:deprecation.py:506 From /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
                                       Instructions for updating:
                                       If using Keras pass *_constraint arguments to layers.
========================================================================= warnings summary ==========================================================================
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/google/protobuf/descriptor.py:47
  /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/google/protobuf/descriptor.py:47: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from google.protobuf.pyext import _message

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/botocore/vendored/requests/packages/urllib3/_collections.py:1
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/botocore/vendored/requests/packages/urllib3/_collections.py:1
  /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/botocore/vendored/requests/packages/urllib3/_collections.py:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Mapping, MutableMapping

-- Docs: https://docs.pytest.org/en/latest/warnings.html
==================================================== 1 failed, 3 passed, 1 skipped, 3 warnings in 16.24 seconds =====================================================
```

@ssaishruthi could you take a look? Thanks!

--- a/tensorflow_addons/metrics/multilabel_confusion_matrix_test.py
+++ b/tensorflow_addons/metrics/multilabel_confusion_matrix_test.py
@@ -13,6 +13,7 @@
 # limitations under the License.
 # ==============================================================================
 """Tests for Multilabel Confusion Matrix Metric."""
+import unittest
 
 import tensorflow as tf
 from tensorflow_addons.metrics import MultiLabelConfusionMatrix
@@ -21,6 +22,7 @@ from tensorflow_addons.utils import test_utils
 
 @test_utils.run_all_in_graph_and_eager_modes
 class MultiLabelConfusionMatrixTest(tf.test.TestCase):
+    @unittest.skip("Failing. See https://github.com/tensorflow/addons/issues/1204")
     def test_config(self):
         mcm_obj = MultiLabelConfusionMatrix(num_classes=3)
         self.assertEqual(mcm_obj.num_classes, 3)
@@ -154,3 +156,7 @@ class MultiLabelConfusionMatrixTest(tf.test.TestCase):
                     [[8, 0], [0, 2]],
                 ],
             )
+
+
+if __name__ == "__main__":
+    tf.test.main()

--- a/tensorflow_addons/metrics/multilabel_confusion_matrix_test.py
+++ b/tensorflow_addons/metrics/multilabel_confusion_matrix_test.py
@@ -13,7 +13,6 @@
 # limitations under the License.
 # ==============================================================================
 """Tests for Multilabel Confusion Matrix Metric."""
-import unittest
 
 import tensorflow as tf
 from tensorflow_addons.metrics import MultiLabelConfusionMatrix
@@ -22,8 +21,8 @@ from tensorflow_addons.utils import test_utils
 
 @test_utils.run_all_in_graph_and_eager_modes
 class MultiLabelConfusionMatrixTest(tf.test.TestCase):
-    @unittest.skip("Failing. See https://github.com/tensorflow/addons/issues/1204")
     def test_config(self):
+        self.skipTest("Failing. See https://github.com/tensorflow/addons/issues/1204")
         mcm_obj = MultiLabelConfusionMatrix(num_classes=3)
         self.assertEqual(mcm_obj.num_classes, 3)
         self.assertEqual(mcm_obj.dtype, tf.int32)

